packages/tools/lint-package/src/rules/api-extractor/types.ts

```
@@ -0,0 +1,27 @@
+import type { Rule } from '../../types.js';
+
+import { accessSync, constants } from 'fs';
+import { join } from 'path';
+
+const rule: Rule = ({ dir, file, pkg }) => {
+  const issues: string[] = [];
+  const filePath = join(dir, file);
+
+  try {
+    accessSync(filePath, constants.F_OK);
+
+    const name = pkg.name?.replace(/.+\//, '');
+    const types = `dist/${name}.d.ts`;
+    if (!pkg.types || pkg.types !== types) {
+      issues.push(
+        `API Extractor is enabled, "types" in package.json should be "${types}"`,
+      );
+    }
+  } catch (error) {
+    // console.warn('Missing config/api-extractor.json');
```

Why do we swallow all errors here?

---

Maybe add a comment indicating that

---

Alright, added a comment